src/protagonist/API/Features/Queues/CustomerQueueController.cs

```diff
@@ -182,10 +188,13 @@ private  void UpdateMembers(int customerId, IList<DLCS.HydraModel.Image>? member
         {
             return this.ValidationFailed(validationResult);
         }
+        
+        var assetsBeforeProcessing = images.Members!

```

Comment:
These 3 lines are identical to those @ 107 - is it worth encapsulating to have a single place to change?